### ISSUE TITLE
root user not properly locked on tiny-ec2-bootstrap calls

### ISSUE THREAD

**Issue by thresheek** [mcrute/tiny-ec2-bootstrap#22]
Looking at the boot output:
```
...
 * Locking root account .../lib/rc/sh/openrc-run.sh: line 102: _lock_root_account: not found
...
```
I guess there is a typo, and `_lock_root_user` (as per https://github.com/mcrute/tiny-ec2-bootstrap/blob/master/tiny-ec2-bootstrap#L88) should be called instead.

**Comment by tomalok** [comment]
Thanks for catching this, we will have this fixed soon.

**Comment by mcrute** [comment]
Fixed this and pushed release-1.4.4. Will get this packaged for aports later today.
